python-sdk/src/astro/sql/operators/cleanup.py

```diff
@@ -224,8 +224,6 @@ def _get_executor_from_job_id(job_id: int) -> str | None:
 
         with create_session() as session:
             job = session.get(Job, job_id)
-        if job.executor_class is None and job.executor:
-            return type(job.executor).__name__
```

Comment:
Using job.executor in this context appears incorrect because it will not only retrieve the executor from the job but also consider the executor conf.

Reply:
I think we should update our test case instead.